run_all_trials.bash

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+TEAM_NAME=$1
+
+#TODO: get list of trials automatically
+declare -a TRIAL_NAMES=("example_trial1" "example_trial2")
+for TRIAL_NAME in ${TRIAL_NAMES[@]}; do
+  echo "Running trial: ${TRIAL_NAME}"
+  #TODO: GET THIS TO RUN THE INDIVIDUAL TRIALS CORRECTLY.
```

Comment:
when the `run_trial.bash` script gets called from here, inside the `run_trial.bash` script the `TRIAL_NAME` variable sometimes has the correct value, but sometimes is empty. (whereas if you call the script from the terminal, the variable is always correct). I didn't have heaps of time to investigate it thoroughly yet - maybe your suggestion to not use bash arrays is enough to fix it.